java-squid/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -129,8 +132,12 @@ public SourceCodeSearchEngine getIndex() {
 
   /**
    * Helper method for testing checks without having to deploy them on a Sonar instance.
+   * Can be dropped when support for CheckMessageVerifier will be dropped.
+   *
+   * @deprecated As of release 3.6, should use {@link org.sonar.java.checks.verifier.JavaCheckVerifier#verify(String filename, JavaFileScanner check)} for rules unit tests.
    */
   @VisibleForTesting
+  @Deprecated
   public static SourceFile scanSingleFile(File file, VisitorsBridge visitorsBridge) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)